app/reducers/ln.js

```diff
@@ -242,12 +252,36 @@ export default function ln(state = {}, action) {
         getNodeInfoAttempt: false,
         nodeInfo: action.nodeInfo
       };
-      case LNWALLET_GETNODEINFO_FAILED:
+    case LNWALLET_GETNODEINFO_FAILED:
       return {
         ...state,
         getNodeInfoAttempt: false,
         nodeInfo: action.error
       };
+    case LNWALLET_GETROUTESINFO_ATTEMPT:
+      return {
+        ...state,
+        getRoutesInfoAttempt: true,
+        routes: null
+      };
+    case LNWALLET_GETROUTESINFO_SUCCESS:
+      return {
+        ...state,
+        getRoutesInfoAttempt: false,
+        routes: action.routes
```

Comment:
You need to set `nodeInfo: null` otherwise if  `LNWALLET_GETROUTESINFO_FAILED` is triggered once `nodeInfo` isn't cleared of the error and the route is never displayed.